src/httprpc.cpp

```diff
@@ -239,12 +239,43 @@ static bool HTTPReq_JSONRPC(const std::any& context, HTTPRequest* req)
     return true;
 }
 
+static std::optional<unsigned> StringToOctal(const std::string& str)
+{
+    unsigned ret = 0;
+    for (char c : str) {
+        if (c < '0' || c > '7') return std::nullopt;
+        ret = (ret << 3) | (c - '0');
+    }
+    return ret;
+}
+
+static std::optional<unsigned> ConvertPermsToOctal(const std::string& str)
+{
+    // Padding the front with 0 sets no special modes if not provided
+    if (str.length() == 3) return StringToOctal("0" + str);
```

Comment:
I don't think we need to prepend the zero here.

Reply:
Ok removed the 0 padding in 1b0fa12fa5